learn/wavemaker-release-notes/v10-10-2.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "WaveMaker 10.10.2 - Release date: 16 December 2021"
+id: ""
+sidebar_label: "Release v10.10.2"
+---
+*WaveMaker released a new update version with critical bug fixes.*
+
+---
+
+## Bug Fixes
+
+- Updated the Spring and Log4j2 dependencies in the platform code base for the reported vulnerabilities.
```

Comment:
Can we change it to "Upgraded spring and Log4j2 dependency versions to mitigate against the vulnerabilities [CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228) and [CVE-2021-22096](https://nvd.nist.gov/vuln/detail/CVE-2021-22096)"